examples/Offline.ClientOnly-sample.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<edmx:Edmx Version="4.0" xmlns:edmx="http://docs.oasis-open.org/odata/ns/edmx">
+  <edmx:Reference Uri="https://sap.github.io/odata-vocabularies/vocabularies/Offline.xml">
+    <edmx:Include Namespace="com.sap.vocabularies.Offline.v1" Alias="Offline" />
+  </edmx:Reference>
+  <edmx:DataServices>
+    <Schema Namespace="clientonly.sample" xmlns="http://docs.oasis-open.org/odata/ns/edm" Alias="client">
+
+      <EntityType Name="SalesOrderType">
+        <Key>
+          <PropertyRef Name="SalesOrder" />
+        </Key>
+        <Property Name="SalesOrder" Type="Edm.String" Nullable="false" MaxLength="32" />
+        <!-- ... and many more -->
+      </EntityType>
+
+      <EntityContainer Name="container">
+        <EntitySet Name="SalesOrder" EntityType="client.SalesOrderType" />
+      </EntityContainer>
+
+      <Annotations Target="client.SalesOrderType">
+        <Annotation Term="Offline.ClientOnly" />
+      </Annotations>
+
+      <Annotations Target="client.SalesOrder">
```

Comment:
If this annotation is targeting the entity set `SalesOrder`, the `Target` has to be
```suggestion
      <Annotations Target="client.container/SalesOrder">
```